release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3178,6 +3178,8 @@ In the following tables, features are marked with the following statuses:
 
 * Enabling LUKS encryption on a system using 512 emulation disks causes provisioning to fail and the system launches the emergency shell in the initramfs. This happens because of an alignment bug in `sfdisk` when growing a partition. As a workaround, you can use Ignition to perform the resizing instead. (link:https://issues.redhat.com/browse/OCPBUGS-35410[*OCPBUGS-35410*])
 
+* {product-title} {product-version} disconnected installation fails on {ibm-power-name} Virtual Server.
```

Comment:
```suggestion
* {product-title} version {product-version} disconnected installation fails on {ibm-power-name} Virtual Server.
```

Reply:
Is there an issue or a workaround to add to this?

Reply:
the build fails everytime when I add the issue link as mentioned below. 

```
* {product-title} {product-version} disconnected installation fails on {ibm-power-name} Virtual Server. 
(link:https://issues.redhat.com/browse/OCPBUGS-36250[*OCPBUGS-36250*])
```

Reply:
fyi : @maxwelldb could you please suggest what can be done in this case.

Reply:
>Is there an issue or a workaround to add to this?
@maxwelldb I have now added the issue link.